cg/cli/deliver/base.py

```diff
@@ -142,19 +142,17 @@ def deliver_ticket(
     """
     cg_context: CGConfig = context.obj
     deliver_ticket_api = DeliverTicketAPI(config=cg_context)
-    is_upload_needed = deliver_ticket_api.check_if_upload_is_needed(ticket_id=ticket_id)
+    is_upload_needed = deliver_ticket_api.check_if_upload_is_needed(ticket=ticket)
     if is_upload_needed:
         LOG.info("Delivering files to customer inbox on hasta")
```

Comment:
I think using "the delivery server" and "the HPC" is good enough. I don't think making them variables improves the readability, especially since doc-strings cannot be f-strings, and we mostly refer to them in doc-strings.